PocketCastsTests/Tests/IAP/IAPHelperTests.swift

```diff
@@ -5,40 +5,30 @@ import StoreKitTest
 import PocketCastsServer
 
 final class IAPHelperTests: XCTestCase {
+    let configurationFile = "Pocket Casts Configuration"
+    let iapTestTimeout: TimeInterval = 5
 
-    var email: String?
+    var session: SKTestSession!
+    var helper: IAPHelper!
 
     override func setUpWithError() throws {
-        // Pretend we're logged in
-        email = ServerSettings.syncingEmail()
-        ServerSettings.setSyncingEmail(email: "test@test.com")
+        session = try SKTestSession(configurationFileNamed: configurationFile)
+        session.clearTransactions()
+        session.resetToDefaultState()
+        session.disableDialogs = true
+
+        helper = IAPHelper(serverHandler: MockIAPHandler())
+        SKPaymentQueue.default().add(helper)
```

Comment:
The payment queue isn't technically used in both of the tests, but if we add more tests then it'll most likely be needed again.